src/main/java/org/plumelib/util/ReflectionPlume.java

```diff
@@ -370,15 +371,14 @@ public static Method methodForName(
 
   // TODO: make this restore the access to its original value, such as private?
   /**
-   * Sets the given field, which may be final and/or private. Leaves the field accessible. Intended
-   * for use in readObject and nowhere else!
+   * Sets the given field, which may be final and/or private. Leaves the field accessible.
    *
-   * @param o object in which to set the field
+   * @param o object in which to set the field; null iff the field is static
    * @param fieldName name of field to set
-   * @param value new value of field
+   * @param value new value of field; may be null iff the field is nullable
    * @throws NoSuchFieldException if the field does not exist in the object
    */
-  public static void setFinalField(Object o, String fieldName, @Nullable Object value)
+  public static void setFinalField(Object o, String fieldName, @Interned Object value)
```

Comment:
The parameter descriptions says "null iff the field is static". So why did the `@Nullable` get removed?
Will most callers pass `@Interned` arguments? Should instead the interned warnings be suppressed?

Reply:
Oh, so both `o` and `value` should be nullable and then you'll need to also suppress nullness warnings.